packages/typescript-estree/src/parseSettings/createParseSettings.ts

```diff
@@ -44,6 +44,7 @@ export function createParseSettings(
         : Array.isArray(options.debugLevel)
         ? new Set(options.debugLevel)
         : new Set(),
+    errorOnInvalidAST: options.errorOnInvalidAST === true,
```

Comment:
Hmmm. I get why we probably want to do opt-in rather than opt-out.. but I wonder if it's better the other way around?

We already throw errors for these cases - so we're turning off the errors with this change.
I think having it opt-out would be best because there's very little reason anyone should want syntactically bad code that (probably) violates the ts-estree types.

WDYT?

Reply:
Yeah that makes sense. Will do!